paths/graphql/04-query-info.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Grab a Query's Response
+permalink: /graphql/query-info
+next-page: /graphql/another-query
+facilitator: false
+sidebar:
+  nav: "github-graphql"
+main-content: |
+
+  1. Using the response from the query in the last step, fill in this template.
+
+          My most recent PRs are: <br>
```

Comment:
I think yes and no - I think it would definitely be easier for the participants if it's fewer PRs, but if it's only one or two I'm not sure if it gets the point across of a 'standup' which we're trying to emulate. I'm open to changing it.